docs/docs/cmd/spo/homesite/homesite-set.mdx

```diff
@@ -52,73 +64,119 @@ Sets the Home site to the provided site collection url and sets the Viva Connect
 
-### Standard response
+```sh
+m365 spo homesite set -u https://contoso.sharepoint.com/sites/contosoportal --draftMode true
+```
 
-<Tabs>
-  <TabItem value="JSON">
+Set the Home site live (not in draft mode)
 
-  ```json
-  "The Home site has been set to https://contoso.sharepoint.com. It may take some time for the change to apply. Check aka.ms/homesites for details."
-  ```
+```sh
+m365 spo homesite set -u https://contoso.sharepoint.com/sites/contosoportal --draftMode false
+```
 
-  </TabItem>
-  <TabItem value="Text">
+Set the audience to everyone
 
-  ```text
-  The Home site has been set to https://contoso.sharepoint.com. It may take some time for the change to apply. Check aka.ms/homesites for details.
-  ```
+```sh
+m365 spo homesite set -u https://contoso.sharepoint.com/sites/contosoportal --targetedLicenseType "everyone"
+```
 
-  </TabItem>
-  <TabItem value="CSV">
+Set the audience to frontline workers
 
-  ```csv
-  The Home site has been set to https://contoso.sharepoint.com. It may take some time for the change to apply. Check aka.ms/homesites for details.
-  ```
+```sh
+m365 spo homesite set -u https://contoso.sharepoint.com/sites/contosoportal --targetedLicenseType "frontLineWorkers"
+```
 
-  </TabItem>
-  <TabItem value="Markdown">
+Set the audience to information workers
 
-  ```md
-  The Home site has been set to https://contoso.sharepoint.com. It may take some time for the change to apply. Check aka.ms/homesites for details.
-  ```
+```sh
+m365 spo homesite set -u https://contoso.sharepoint.com/sites/contosoportal --targetedLicenseType "informationWorkers"
+```
 
-  </TabItem>
-</Tabs>
+Set the audience to M365 groups using M365 Groups Ids
+
+```sh
+m365 spo homesite set -u https://contoso.sharepoint.com/sites/contosoportal --audienceIds "978b5280-4f80-47ea-a1db-b0d1d2fb1ba4,21af775d-17b3-4637-94a4-2ba8625277cb"
+```
 
-### `vivaConnectionsDefaultStart` response
+Set the audience to M365 groups using M365 Groups Names
 
-When we make use of the option `vivaConnectionsDefaultStart` the response will differ. 
+```sh
+m365 spo homesite set -u https://contoso.sharepoint.com/sites/contosoportal --audienceNames "Marketing, IT department"
+```
+
+## Response
+
+### Standard response
 
 <Tabs>
   <TabItem value="JSON">
 
   ```json
-  "The Home site has been set to https://contoso.sharepoint.com and the Viva Connections default experience to True. It may take some time for the change to apply. Check aka.ms/homesites for details."
+  {
+  "Audiences": [
+    {
+      "Email": "active@contoso.onmicrosoft.com",
+      "Id": "7a1eea7f-9ab0-40ff-8f2e-0083d9d63451",
+      "Title": "active Members"
+    }
+  ],
+  "IsInDraftMode": true,
+  "IsVivaBackendSite": false,
+  "SiteId": "431d7819-4aaf-49a1-b664-b2fe9e609b63",
+  "TargetedLicenseType": 2,
+  "Title": "The Landing",
+  "Url": "https://contoso.sharepoint.com/sites/TheLanding",
+  "VivaConnectionsDefaultStart": true,
+  "WebId": "626c1724-8ac8-45d5-af87-c07c752fab75"
```

Comment:
I think this response it not well aligned. The property names are on the same level as the brackets. Let's give them one indent.